database/database.go

```diff
@@ -299,10 +397,22 @@ func UpdateQuote(q QuoteT) error {
 		`UPDATE quotes SET TeacherID=$2, Context=$3, Text=$4 WHERE QuoteID=$1`,
```

Comment:
You can specify the error text further - you're already doing the strinf compare, after all